src/service/poc_lora.proto

```diff
@@ -167,21 +167,32 @@ message lora_poc_v1 {
   repeated lora_verified_witness_report_v1 unselected_witnesses = 4;
 }
 
-// iot per-gateway reward share for a given reward period
-// for PoC coverage events provided between the start and end timestamps
-message gateway_reward_share {
+message gateway_reward {
   /// Public key of the hotspot
   bytes hotspot_key = 1;
   /// Amount credited to the hotspot for beaconing
   uint64 beacon_amount = 2;
   /// Amount credited to the hotspot for witnessing
   uint64 witness_amount = 3;
+  /// Amount credited to the hotspot for data transfer
+  uint64 dc_transfer_amount = 4;
+}
+
+message operational_reward {
+  /// Amount credited to the operational fund wallet
```

Comment:
maybe declare unit here for the amount?

Reply:
addressed here: 870adbada95b32010d42db966a1dbc55b1b73605